app/src/main/java/ch/epfl/sweng/studyup/MainActivity.java

```diff
@@ -83,7 +77,10 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         displayLoginSuccessMessage(getIntent());
-        Firestore.get().loadQuestions(this);
+
+        Firestore.loadQuestions(this);
```

Comment:
Why not Firestore.get() there?

Reply:
Sorry my mistake, I pushed a commit with, normally, no typos like this anymore